humble/distribution.yaml

```
@@ -1972,7 +1972,7 @@ repositories:
       tags:
         release: release/humble/{package}/{version}
       url: https://github.com/ros2-gbp/mavlink-gbp-release.git
-      version: 2022.2.2-3
+      version: 2022.6.27-1
     source:
       type: git
       url: https://github.com/mavlink/mavlink-gbp-release.git
```

```suggestion
      url: https://github.com/mavlink/mavlink-gbp-release.git
      url: https://github.com/mavlink/mavlink.git
```

Usually the source url is the source repository, but if there are lots of customizations in the release repository to support ROS it would make sense to point people to it.

---

@vooon branch shouldn't be `release/humble/mavlink` though?

---

@hidmic yes, good catch. But actually there is no difference with rolling, at least for now.